db_master_deploy/deploy.sh

```diff
@@ -11,10 +11,11 @@ display_usage() {
     echo -e "Usage:\n$0 -s source_objects -t target_staging -a timestamp for BOD snapshot/archive (YYYYMMDD)"
     echo -e "\t-s comma delimited list of source databases and/or tables - mandatory"
     echo -e "\t-t target staging - mandatory choose one of '${targets}'"
+    echo -e "\t-r materialized view to refresh - Optional. You can define (with dbname.schema.tablename !!) one or more materialized views to refresh'"
```

Comment:
I think it would make more sense to have a on/off switch here, just refresh materialized views true or false.
If true, all materialized views which are based on ``-s tables or database`` will be refreshed. If you have to pass the list of materialized views, you can also refresh them manually in the database first... but that's a bit a matter of perspective...